allensdk/brain_observatory/ecephys/ecephys_session_api/ecephys_session_api.py

```diff
@@ -16,6 +17,9 @@ class EcephysSessionApi:
     def __init__(self, *args, **kwargs):
         pass
 
+    def get_session_start_time(self) -> datetime:
```

Comment:
Maybe these should be `@abstractclass` instead of `NotImplementedError` (or maybe both?) since it looks like `EcephysSessionAPI` is a base class.

Reply:
`EcephysSessionApi` needs to be implementable in order to support static caches. This is the use pattern where someone downloads a bunch of data once and then decides not to change that set of data so that they have something stable to analyze for e.g. a paper.

Reply:
got it, makes sense!